//
// PROJECT:         Aspia
// FILE:            category/category_memory.proto
// LICENSE:         GNU Lesser General Public License 2.1
// PROGRAMMERS:     Dmitry Chapyshev (dmitry@aspia.ru)
//

syntax = "proto3";

option optimize_for = LITE_RUNTIME;

package aspia.proto;

message Memory
{
    uint64 total_physical  = 1;
    uint64 free_physical   = 2;
    uint64 total_page_file = 3;
    uint64 free_page_file  = 4;
    uint64 total_virtual   = 5;
    uint64 free_virtual    = 6;
}
